package.json

```diff
@@ -31,84 +30,39 @@
     "@mdi/font": "^5.9.55",
+    "@vue/cli-plugin-babel": "^3.12.0",
+    "@vue/cli-plugin-eslint": "^3.12.0",
+    "@vue/cli-plugin-unit-mocha": "^3.12.0",
+    "@vue/cli-service": "^3.12.0",
+    "@vue/eslint-config-standard": "^4.0.0",
     "@vue/test-utils": "^1.0.3",
-    "autoprefixer": "^9.8.0",
-    "babel-eslint": "^9.0.0",
-    "babel-loader": "^8.0.0",
-    "babel-plugin-istanbul": "^4.1.5",
+    "babel-eslint": "^10.0.1",
+    "babel-plugin-istanbul": "^6.1.1",
     "chai": "^4.2.0",
-    "chalk": "^2.4.2",
-    "connect-history-api-fallback": "^1.6.0",
     "copy-dir": "^1.2.0",
-    "copy-webpack-plugin": "^4.3.1",
-    "core-js": "^3.9.1",
-    "cross-env": "^5.2.1",
-    "css-loader": "^1.0.1",
-    "cssnano": "^3.10.0",
+    "cross-env": "^7.0.3",
     "eslint": "^5.16.0",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-friendly-formatter": "^3.0.0",
-    "eslint-loader": "^2.2.1",
-    "eslint-plugin-html": "^4.0.6",
-    "eslint-plugin-import": "^2.18.0",
-    "eslint-plugin-node": "^9.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1",
-    "eventsource-polyfill": "^0.9.6",
-    "express": "^4.17.1",
-    "extract-text-webpack-plugin": "^2.0.0",
-    "file-loader": "^2.0.0",
-    "friendly-errors-webpack-plugin": "^1.1.3",
+    "eslint-formatter-friendly": "^7.0.0",
+    "eslint-plugin-vue": "^5.2.3",
     "gh-pages": "^1.1.0",
-    "html-webpack-plugin": "^2.28.0",
-    "http-proxy-middleware": "^0.19.2",
-    "inject-loader": "^4.0.1",
-    "karma": "^4.4.1",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-coverage": "^1.1.1",
-    "karma-mocha": "^1.3.0",
-    "karma-phantomjs-launcher": "^1.0.2",
-    "karma-phantomjs-shim": "^1.5.0",
+    "karma": "^6.4.0",
+    "karma-chrome-launcher": "^3.1.1",
+    "karma-cli": "^2.0.0",
+    "karma-coverage": "^2.2.0",
+    "karma-mocha": "^2.0.1",
     "karma-sinon-chai": "^2.0.2",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-spec-reporter": "^0.0.32",
-    "karma-webpack": "^3.0.5",
+    "karma-sourcemap-loader": "^0.3.8",
+    "karma-spec-reporter": "^0.0.34",
+    "karma-webpack": "^4.0.2",
     "material-icons": "^0.5.4",
-    "mocha": "^6.2.3",
-    "opn": "^6.0.0",
-    "optimize-css-assets-webpack-plugin": "^2.0.0",
-    "ora": "^3.4.0",
-    "phantomjs-prebuilt": "^2.1.16",
     "portal-vue": "^2.1.7",
-    "rimraf": "^2.7.1",
-    "semver": "^6.3.0",
-    "shelljs": "^0.8.4",
-    "sinon": "^7.5.0",
+    "sass": "^1.19.0",
+    "sass-loader": "^8.0.0",
+    "sinon": "^9.2.4",
     "sinon-chai": "^3.5.0",
-    "url-loader": "^0.6.2",
-    "vue-loader": "^14.2.4",
-    "vue-style-loader": "^4.1.2",
-    "vue-template-compiler": "2.6.11",
-    "webpack": "^2.7.0",
-    "webpack-bundle-analyzer": "^3.8.0",
-    "webpack-dev-middleware": "^1.12.2",
-    "webpack-hot-middleware": "^2.25.0",
-    "webpack-merge": "^4.2.2"
-  },
-  "engines": {
-    "node": ">= 4.0.0",
-    "npm": ">= 3.0.0"
-  },
```

Comment:
Could you re-introduce the required min engine requirements again?

Reply:
Certainly !

In fact, latest version of Vue-cli2 upgraded those requirements to node >= 6.0.0 (73881c2eb00704a4e616070deaee072a29280df6) mainly because of Webpack3

Then they completely removed these requirements in Vue-cli3. However, going through the source code of vue-cli, it seems requirements should be node >=8.9 once again, because of Webpack4. npm isn't mentioned anymore as npm is automatically bundled inside node.

Is it O.K. for you to set `  "engines": {
    "node": ">= 8.9"
  },` ?

Reply:
After looking in the Node release notes, I found that npm version bundled with node 8.9.0 is npm 5.5.1 so I eventually propose to set

    "engines": {
        "node": ">= 8.9",
        "npm": ">= 5.5.1"
    },

Reply:
[This site ](https://www.sitepoint.com/vue-cli-intro/ ) states that node 8.9+ is minimal but 8.11.0 is recommended. So probably 
```
"engines": {
    "node": ">= 8.11",
    "npm": ">= 5.6.0"
},
```
is a safe choise.

Reply:
For completeness, [this](https://github.com/vuejs/vue-cli/blob/v3.12.1/docs/guide/installation.md) is the text from the official Vue-CLI site recommending 8.11.0.

This is  resolved by [dad49c4](https://github.com/wegue-oss/wegue/pull/297/commits/dad49c4fb325fb6a96616104a67b877b5dae0378)